components/card/card.scss

```diff
@@ -0,0 +1,33 @@
+.Card {
+    border: 1px solid var(--center-channel-color-08);
```

Comment:
Should we be using the new `rgba(var(--center-channel-color-rgb), 0.8)` approach here? (might have the exact syntax wrong as I'm going on memory atm)

Reply:
Ahh I keep using this wrong! Will fix.